docs/docs/cmd/spo/applicationcustomizer/applicationcustomizer-add.md

```diff
@@ -22,6 +22,9 @@ m365 spo applicationcustomizer add [options]
 `--clientSideComponentProperties [clientSideComponentProperties]`
 : JSON string with application customizer properties
 
+`-s, --scope [scope]`
+: Scope of the application customizer. Allowed values: `Site`, `Web`. Defaults to `Site`.
```

Comment:
@martinlingstuyl I know you have a lot but did you maybe managed to take a look at this one?

Reply:
Sorry, didn’t see the notification @Adam-it! 
I chose Site as default in the spec because Site VS Web is kind of an old SP architecture distinction. subwebs are now out of fashion and site collections are just called Sites these days. So let’s align with this modern way of thinking and default to Site.

Reply:
still will it be consistent with what we already have? maybe we should have an issue to update the `customaction add` command as well 🤔?

Reply:
Good point, it would be a breaking change though... let's at least start by using site here.

Reply:
Yes it will, but I would like us to be co consistent in the approach. If we suddenly want to change it now. Let's also plan to fix it for what we already have.

Reply:
Do @nicodecleyre, you already fixed it for this one?

Reply:
No, I've changed it to Web in [temp commit](https://github.com/pnp/cli-microsoft365/commit/4c044f4315e18a29e888c49169728fb315c37b38). But given our last conversation, I'll change it back to Site in this one

Reply:
@nicodecleyre first of all sorry for the late reply. I know you are doing a lot a lot of awesome work 👍.
I tried to rereview this one to make it ready to be merged for the next preview (next month).
I have question if you maybe had a chance to revert the last changes and changed to `Site` as @martinlingstuyl suggested? TBH I checked and I think it is still `Web`

Reply:
Right,... 😄 seems I lost track about that one! Sorry about that! Thank you for following this up! Updated the PR